ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/input-variables.md

```diff
@@ -321,6 +321,12 @@ ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/interface-pr
 ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/lacp.md
 --8<--
 
+### Load Balance
```

Comment:
The policies are then applied to port-channel key so it probably makes sense to put it here but let's think if this is the best place.

Reply:
let's move it to the System Settings section with other hardware settings